### DISCUSSION TITLE
Is there a way to use an `EXCLUDE_FILES` list?

### DISCUSSION THREAD

**Discussion by behudson** [super-linter/super-linter#5199] [Q&A] [answered]
I've seen the `FILTER_REGEX_EXCLUDE`, but it doesn't seem to support multiple regular expressions and does not allow exclusions of varying paths/filenames. Is there a way to support either a list of regular expressions or a simple list of files to exclude?

**Comment by ferrarimarco** [comment, marked as answer]
Hi @behudson !

Both `FILTER_REGEX_EXCLUDE` and `FILTER_REGEX_INCLUDE` are Bash regular expression, so you should be able to craft a "complex" regex that suits your needs. At this point, there's no support for specifying multiple regular expressions.

You can also check if the linters you're interested in support an ignore mechanism themselves, and use that instead of `FILTER_REGEX_EXCLUDE` and `FILTER_REGEX_INCLUDE`.

**Reply by ferrarimarco** [reply]
Example: `.*(/test/linters/|CHANGELOG.md).*`

**Reply by behudson** [reply]
Thank you for the example. I came up with something similar that seemed to do the trick. Attempting to use the individual linters proved problematic as the different Python linters did not behave well together. 
Example: `"(^.*file1.*$)|(^.*path1/file2.*$)"`
